extensions/wikia/SpecialDiscussionsLog/search/IpAddressQuery.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Wikia\SpecialDiscussionsLog\Search;
+
+
+class IpAddressQuery implements SearchQuery {
+
+	private $searchKey;
+
+	public function __construct( $searchKey ) {
+		$this->searchKey = $searchKey;
+	}
+
+	function getKey() {
```

Comment:
`IpAddressQuery` and `UserQuery` no longer need `$searchKey`, `__construct()`, or `getKeyName()`.